alicloud/common.go

```diff
@@ -698,6 +698,14 @@ func ParseResourceId(id string, length int) (parts []string, err error) {
 	return parts, err
 }
 
+func ParseSlbListenerId(id string) (parts []string, err error) {
+	parts = strings.Split(id, ":")
+	if len(parts) != 2 && len(parts) != 3 {
+		err = WrapError(fmt.Errorf("Invalid alicloud_slb_listener Id %s.", id))
```

Comment:
Please add more message to guide users to set the correct resource id, like line 696.